translations/ui-agreements/en_GB.json

```diff
@@ -220,5 +220,6 @@
     "license.unlinkLicense": "Unlink license",
     "license.noLicenses": "Agreement has no licenses.",
     "agreements.internalContactsRole": "Internal contact role",
-    "eresources.defaultCoverage": "Default coverage"
+    "eresources.defaultCoverage": "Default coverage",
+    "customcoverages.tooltip": "Custom Coverage"
```

Comment:
Setting it doesn't make a real difference, but you don't need to specify translation keys for languages other than `en`. They'll eventually be run through and generated.

Reply:
Ok, should I leave those for now anyway, or go through and remove?

Reply:
Nah, leave em. Just trying to save you time for the future.